cognite/_utils.py

```diff
@@ -27,6 +27,22 @@ def _status_is_valid(status_code: int):
 def _should_retry(status_code):
     return status_code in [401, 429, 500, 502, 503]
 
+def serialize(obj):
+    """JSON serializer for objects not serializable by default json code"""
+    return obj.__dict__
+
+
+def delete_request(url, params=None, headers=None, cookies=None):
```

Comment:
delete_request already exists. remove this one.

Reply:
done in https://github.com/cognitedata/cognite-sdk-python/pull/150/commits/5b37dc40cee64f385cf2e41290c54cd0d94c2358